.yamllint.yml

```diff
@@ -1,7 +1,20 @@
 ---
+# Based on https://github.com/kubernetes-sigs/kubespray
 extends: default
 
+ignore: |
+  .git/
+
 rules:
```

Comment:
The RedHat CoP project has a more extensive linting that goes into more details:

https://redhat-cop.github.io/ci/linting-testing-helm-charts.html

> This file enforces style standards across CoP projects. Ct uses this file to lint the values.yaml and Chart.yaml files. Below is the recommended lintconf.yaml config.